opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -312,6 +314,9 @@ namespace cpgrid
                 }
             }
 
+            Opm::OpmLog::info(fmt::format("{} pinch-out connections generated",
+                                          nnc_cells[PinchNNC].size()));
```

Comment:
So this will print

> 0 pinch-out connections generated

if there are no additional connections.  That's okay, if a little redundant.  Worse, however, is that it will print

> 1 pinch-out connection**S** generated

i.e., a plural, for the case of a single pinch-out connection.  That's not ideal and there's no reason to do that here because we can easily query the number of connections emit a message in concord in this case.